modules/oe_content_news/oe_content_news.info.yml

```diff
@@ -10,6 +10,9 @@ dependencies:
   - oe_content:oe_content_reference_code_field
   - oe_media:oe_media
   - oe_media_avportal:oe_media_avportal
+  - oe_content_entity_contact:oe_content_entity_contact
+  - composite_reference:composite_reference
+  - entity_reference_revisions:entity_reference_revisions
```

Comment:
`inline_entity_form` is also required now. Also update readme for the submodule, including possible patches.

Reply:
Updated and readme added